forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/main/java/com/forgerock/openbanking/aspsp/rs/store/api/openbanking/event/v3_0/CallbackUrlsApiController.java

```diff
@@ -35,26 +34,45 @@
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestHeader;
-import uk.org.openbanking.datamodel.event.*;
+import uk.org.openbanking.datamodel.event.OBCallbackUrl1;
 
 import javax.servlet.http.HttpServletRequest;
 import javax.validation.Valid;
 import java.security.Principal;
 import java.util.Collection;
 import java.util.Optional;
 import java.util.UUID;
-import java.util.stream.Collectors;
 
 @Controller("CallbackUrlsApiV3.0")
 @Slf4j
 public class CallbackUrlsApiController implements CallbackUrlsApi {
 
     private CallbackUrlsRepository callbackUrlsRepository;
     private TppRepository tppRepository;
+    private EventResponseUtilService eventResponseUtilService;
 
-    public CallbackUrlsApiController(CallbackUrlsRepository callbackUrlsRepository, TppRepository tppRepository) {
+    public CallbackUrlsApiController(CallbackUrlsRepository callbackUrlsRepository, TppRepository tppRepository, EventResponseUtilService eventResponseUtilService) {
         this.callbackUrlsRepository = callbackUrlsRepository;
         this.tppRepository = tppRepository;
+        this.eventResponseUtilService = eventResponseUtilService;
```

Comment:
Rather than having it as a constructor injected bean, you could initialise it here and then do away with the `initialiseResponseUtil()` method below and the multiple calls to it. The problem with the latter is that it would be easy to forget to call it.